guides/source/action_view_helpers.md

```diff
@@ -507,166 +717,328 @@ number_with_precision(111.2345)               # => 111.235
+If either the `:attributes` or `:tags` options are passed, only the mentioned
+attributes and tags are allowed and nothing else.
 
 ```ruby
 sanitize @article.body, tags: %w(table tr td), attributes: %w(id class style)
 ```
 
-To change defaults for multiple uses, for example adding table tags to the default:
+To change defaults for multiple uses, for example, adding table tags to the
+default:
 
 ```ruby
 class Application < Rails::Application
-  config.action_view.sanitized_allowed_tags = 'table', 'tr', 'td'
+  config.action_view.sanitized_allowed_tags = %w(table tr td)
 end
 ```
 
-See the [API Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-sanitize) for more information.
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-sanitize)
+for more information.
+
+#### sanitize_css
 
-#### sanitize_css(style)
+Sanitizes a block of CSS code, particularly when it comes across a style
+attribute in HTML content. `sanitize_css` is particularly useful when dealing
+with user-generated content or dynamic content that includes style attributes.
 
-Sanitizes a block of CSS code.
+The `sanitize_css` method below will remove the styles that are not allowed.
 
-See the [API Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-sanitize_css) for more information.
+```ruby
+sanitize_css("background-color: red; color: white; font-size: 16px;")
+```
 
-#### strip_links(html)
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-sanitize_css)
+for more information.
+
+#### strip_links
 
 Strips all link tags from text leaving just the link text.
 
 ```ruby
 strip_links('<a href="https://rubyonrails.org">Ruby on Rails</a>')
 # => Ruby on Rails
-```
 
-```ruby
 strip_links('emails to <a href="mailto:me@email.com">me@email.com</a>.')
 # => emails to me@email.com.
-```
 
-```ruby
 strip_links('Blog: <a href="http://myblog.com/">Visit</a>.')
 # => Blog: Visit.
 ```
 
-See the [API Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-strip_links) for more information.
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-strip_links)
+for more information.
 
-#### strip_tags(html)
+#### strip_tags
 
-Strips all HTML tags from the html, including comments.
-This functionality is powered by the rails-html-sanitizer gem.
+Strips all HTML tags from the HTML, including comments.
 
 ```ruby
 strip_tags("Strip <i>these</i> tags!")
 # => Strip these tags!
+
+strip_tags("<b>Bold</b> no more! <a href='more.html'>See more</a>")
+# => Bold no more! See more
 ```
 
+NOTE: The output may still contain unescaped `<`, `>`, `&` characters and
+confuse browsers.
+
```

Comment:
I think this can be removed.
After [this PR](https://github.com/rails/rails/pull/28061) all special characters will be escaped. The PR also removed it from the [documentation.](https://github.com/rails/rails/pull/28061/files#diff-db13dbc376db15388ddbcb4c6c220b4339462c05e360e809c983ff60da3c862c)
```suggestion
```

Reply:
✅ + updated the description to read `Strips all HTML tags from the HTML, including comments and special characters.` and added a malformed example.